src/modules/fancyzones/lib/FancyZones.cpp

```diff
@@ -940,6 +947,18 @@ bool FancyZones::IsNewWorkArea(GUID virtualDesktopId, HMONITOR monitor) noexcept
     return true;
 }
 
+bool FancyZones::IsSplashScreen(HWND window)
+{
+    wchar_t splashString[] = L"MsoSplash";

```

Comment:
Add a comment to clarify this string should not be localized. I know it may sound silly, but localization may be done by someone that doesn't have any specific knowledge of this code. Also since it's a class name, make that explicit in the string name.